deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/assigner
+  - ingress.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+
```

Comment:
We are missing `configMapGenerator` here, which generates `config` resource that is then specified as a volume in deployment patch; i.e.:

```
     volumes:
         - name: config
           configMap:
             name: config
```
Read as: create a volume named config, the content should come from a configmap named `config`.

Try comparing the output of `kustomize build .` before and after adding `configMapGenerator` to see how it impacts the generated manifests.

Reply:
Done.

Server dry run:
```sh
kustomize build . | kubectl -n storetheindex apply --dry-run=server -f -
configmap/config-b86gc24tb4 created (server dry run)
secret/identity-b6746b8676 created (server dry run)
service/assigner created (server dry run)
deployment.apps/assigner created (server dry run)
ingress.networking.k8s.io/assigner created (server dry run)
```
Client dry-run
```sh
kustomize build . | kubectl -n storetheindex apply --dry-run=client -f -
configmap/config-b86gc24tb4 created (dry run)
secret/identity-b6746b8676 created (dry run)
service/assigner created (dry run)
deployment.apps/assigner created (dry run)
ingress.networking.k8s.io/assigner created (dry run)
```